bin/select_pseudo_irts_from_lib.py

```diff
@@ -71,6 +90,12 @@ def main():
         help='maximum rt of irts to select for alignment'
     )
 
+    model.add_argument(
+        '-q', '--quantiles',
+        type=str,
+        help='whether to use only the 1st and 4th RT quantile for irt selection'
```

Comment:
add what strings are valid. e.g., if someone puts True it will fail because it is expecting true?

Reply:
So the parameter is already to boolean by nextflow and is then converted into strings "true" or "false" within the workflow. Just in the python script it is not restricted, but I can still add it.

Reply:
Ideally you'd use a proper boolean flag here in the python argparse and skip the bool string thingy in nextflow along the lines of

```
[...]
script:
quant_flag = params.irts_from_outer_quantiles ? "--quantiles" : ""
"""
select_pseudo_irts_from_lib.py ${quant_flag} [...]
"""
```

Reply:
For core readability it's also a bit confusing that the variable is sometimes called `quantile` and sometimes `quantiles`.

Reply:
done